agenta-web/src/lib/helpers/evaluate.ts

```diff
@@ -0,0 +1,76 @@
+import {convertToCsv, downloadCsv} from "./utils"
+
+export const exportExactEvaluationData = (evaluation: any, rows: any[]) => {
+    const exportRow = rows.map((data) => {
+        return {
+            ["Inputs"]: data.inputs[0].input_value,
+            ["App Variant v0 Output"]: data?.columnData0
```

Comment:
this is a hardcoded value. the name of the variant is variable.
In the example below I used v1 and in the exported file it's v0.
><img width="1066" alt="Screenshot 2023-09-13 at 06 16 17" src="https://github.com/Agenta-AI/agenta/assets/6608260/0bb9964c-4aa1-49c1-b5d4-f1193afdade0">
><img width="520" alt="Screenshot 2023-09-13 at 06 18 43" src="https://github.com/Agenta-AI/agenta/assets/6608260/62e13e3f-81a9-438a-a267-c2785d9c7090">

Reply:
Fixed